db/indexCass.js

```diff
@@ -0,0 +1,39 @@
+const cassandra = require('cassandra-driver');
+
+const client = new cassandra.Client({
+  contactPoints: ['127.0.0.1'],
+  keyspace: 'testyboi',
+  localDataCenter: 'datacenter1'
+})
```

Comment:
I would recommend configuring your eslint config for Airbnb's style guide so you can be more consistent w/ your usage of semi colons